gobblin-runtime/src/main/java/gobblin/scheduler/JobScheduler.java

```diff
@@ -145,7 +145,7 @@ protected void startUp() throws Exception {
 
     this.scheduler.start();
     if (this.properties.containsKey(ConfigurationKeys.JOB_CONFIG_FILE_DIR_KEY)) {
```

Comment:
It seems you are checking for the presence of `jobconf.dir`, then using `jobconf.fullyQualifiedDir` to actually get the job conf location, and if `jobconf.dir` is not present, it won't run anything, am I right?
The correct logic should be:
* if `jobconf.fullyQualifiedDir` is present, run `scheduleGeneralConfiguredJobs`.
* else if `jobconf.dir` is present, then set `jobconf.fullyQualifiedDir` to `file:///<jobconf.dir>`
* else throw an error